### ISSUE TITLE
DFL 2022 | Resources - Article link block

### ISSUE THREAD

**Issue by kevinhowbrook** [mozilla/foundation.mozilla.org#9401]
### Describe the feature request
New block for an article link, should consist of the following:
- Article (page chooser)

### Screenshots / mock-ups

![Screenshot(9)](https://user-images.githubusercontent.com/2128707/191293313-14e15e2b-1c5d-48fc-ab84-af7f938c53cf.png)

### Additional context
Link to [designs](https://www.figma.com/file/goD9bRqPoedQAxi8OsInkR/Data-Futures-Lab%3A-Designs?node-id=644%3A7950)

The title and author should be pulled from the selected article page.  Similar elements exist in the news section on the homepage.

---

**Dev Tasks:** 
* [ ] Create new article link  block consisting of a page chooser for articles
* [ ] Configure article link block template to pull information from the selected page

**Acceptance Criteria:**
- [ ] Design to look as per Figma files
- [ ] As an editor creating a banner page I can add an article link grid block consisting of 3 links to article pages
- [ ] Template stacks verticle on mobile

NOTE: To be tested with #9402

**Comment by cdanfon** [comment]
@kristinashu Can you confirm if this component should have blue or black as the colour for the title please? Thanks!

**Comment by kristinashu** [comment]
They should be black, ty!

**Comment by kevinhowbrook** [comment]
PR is here https://github.com/mozilla/foundation.mozilla.org/pull/9521

However it's blocked for the moment, I'm not sure what the circular images are in relation to the article 

![image](https://user-images.githubusercontent.com/2128707/197219668-c57e1cb3-b0e3-4ecf-82c5-7577d3dc1798.png)

**Comment by kevinhowbrook** [comment]
Bens informed me we are pretty sure these are authors so I'll wire those up

**Comment by kevinhowbrook** [comment]
This is ready for review https://github.com/mozilla/foundation.mozilla.org/pull/9521

**Comment by kevinhowbrook** [comment]
This is approved and ready to deploy but we are waiting behind other PRs

**Comment by cdanfon** [comment]
Hey @kevinhowbrook thanks for the update - would this need to be re-code reviewed once the other PRs have been checked? In which case, can we move this to ready to deploy and add the 'blocked' label ?

**Comment by kevinhowbrook** [comment]
This was accidentally merged. New PR is here https://github.com/mozilla/foundation.mozilla.org/pull/9562

It's blocked by https://github.com/mozilla/foundation.mozilla.org/pull/9511

**Comment by tbrlpld** [comment]
@cdanfon Ready for test on staging

**Comment by cdanfon** [comment]
I'm struggling to test this as I can only select article pages but I'm not sure where to find one in the page tree. @tbrlpld can you help? 

@kevinhowbrook 'cc

**Comment by cdanfon** [comment]
Ignore my comment @tbrlpld I've managed to sort it. 

@stevedya the underline for the article header should be black as per styler [guide](https://foundation.mozilla.org/en/style-guide/#headers-links)

**Comment by stevedya** [comment]
@tbrlpld I've added a fix PR for these now and its ready for review here #9565 

cc: @cdanfon

**Comment by tbrlpld** [comment]
Left one small question @stevedya

**Comment by stevedya** [comment]
@tbrlpld No problemo! I've addressed that question and added a couple more improvements that are ready to be looked at 😄

**Comment by tbrlpld** [comment]
@cdanfon Why are you assigning me to this issue? It has been reviewed and deployed to QA yesterday.

**Comment by cdanfon** [comment]
I was looking at this issue in the DFL kanban board and couldn't see the section that says you've merged the pull request - I've assumed you needed to review it after Steve's comment. Thanks Tibor!

**Comment by cdanfon** [comment]
Looks good to me, over to @gutermul for MoFo testing

**Comment by gutermul** [comment]
From the designs and final content it's clear that the intention for these was to be news article selections from the internet- not from within the mofo site, but I see the dev task was to use a page picker. I will raise this with the DFL team on Tuesday to see if we can shift to adapt and get back to you.

**Comment by cdanfon** [comment]
After speaking with @gutermul we've agreed to mark this as completed, however Lisa would need to have a chat with the rest of the DFL team to understand if ticket #9585 needs to be addressed

**Comment by cdanfon** [comment]
This is now on prod. Closing as completed
